packages/web/components/swap-tool/trade-details.tsx

```diff
@@ -100,13 +97,9 @@ export const TradeDetails = ({
                       }
                     )}
                   >
-                    {inDenom} {t("assets.table.price").toLowerCase()} ≈{" "}
-                    {inPrice &&
-                      formatPretty(inPrice ?? inPrice ?? new Dec(0), {
-                        maxDecimals: inPrice
-                          ? 2
-                          : Math.min(swapState?.toAsset?.coinDecimals ?? 8, 8),
-                      })}
+                    {swapState?.inBaseOutQuoteSpotPrice
+                      ? ExpectedRate(swapState, outAsBase, treatAsStable)
+                      : ""}
```

Comment:
Nit: Replace with coalescing operator

```suggestion
                    {swapState?.inBaseOutQuoteSpotPrice &&
                      ExpectedRate(swapState, outAsBase, treatAsStable)}
```